class.js

```diff
@@ -0,0 +1,56 @@
+import { getProductList } from './product.js';
+
+class Product {
+    constructor(name, description, price, tags, images, favoriteCount) {
+        this.name = name;
+        this.description = description;
+        this.price = price;
+        this.tags = tags;
+        this.images = images;
+        this.favoriteCount = favoriteCount + 1;
+    }
```

Comment:
constructor에서 증가시키는 것이 아닌 따로 메서드를 만들어야함.
복사본을 넣어야 원본이 변경되어도 내부 멤버변수가 변하지 않음
```suggestion
    constructor(name, description, price, tags, images, favoriteCount) {
        this.name = name;
        this.description = description;
        this.price = price;
        this.tags = [...tags];
        this.images = [...images];
        this.favoriteCount = favoriteCount;
    }
    
    favorite() {
        this.favoriteCount +=1;
    }
```